test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore/packages.config

```diff
@@ -48,10 +48,10 @@
   <package id="Microsoft.Extensions.Primitives" version="2.0.0" targetFramework="net461" />
   <package id="Microsoft.Net.Http.Headers" version="2.0.1" targetFramework="net461" />
   <package id="Microsoft.NETCore.Platforms" version="1.0.1" targetFramework="net461" />
-  <package id="Microsoft.OData.Client" version="7.4.0-beta" targetFramework="net461" />
```

Comment:
We should probably do this to all other projects, including perf, as well I think.

Reply:
Yes. I am working on this. See my next PR.